### PR TITLE
Created velocity control mode for autonomous paths

### PR DESCRIPTION
**Pull request by ngupta-22**
Updated RamseteFollowerExectute and MoveWithJoysticks to be able to use velocity control mode (added PID, constants, and velocity MPS to native units conversion), added new paths, and updated max speed.

### CODE REVIEW

**Review by cgjyang — APPROVED** [review]
very cool 👍

**Review by yabberyabber — COMMENTED** [review]
Great progress! This stuff is not easy and you're doing a great job staying focused

File: Comp2022/PathWeaver/Paths/forward79 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+X,Y,Tangent X,Tangent Y,Fixed Theta,Reversed,Name
```
[the remark this answers is not in the document]
Comment (ngupta-22) [reply Comp2022/PathWeaver/Paths/forward79:1]: that's the distance of the path in inches

File: Comp2022/src/main/include/Constants.h [hunk 2]
```diff
@@ -34,6 +34,7 @@ namespace DriveConstants
 {
     // Odometry constants
     static constexpr int kEncoderCPR = 2048; // CPR is 2048 for new TalonFX
+    static constexpr int kRPM = 6380;
```
[the remark this answers is not in the document]
Comment (ngupta-22) [reply Comp2022/src/main/include/Constants.h]: Yeah I did! It's the free speed for the Falcon 500

File: Comp2022/src/main/cpp/subsystems/Drivetrain.cpp [hunk 3]
```diff
@@ -530,7 +566,31 @@ void Drivetrain::MoveWithJoysticks(frc::XboxController *throttleJstick)
     }
 
     if (m_talonValidL1 || m_talonValidR3)
-        m_diffDrive.CurvatureDrive(-yOutput, xOutput, m_isQuickTurn);
+    {
+#if 0
```
[the remark this answers is not in the document]
Comment (ngupta-22) [reply Comp2022/src/main/cpp/subsystems/Drivetrain.cpp]: Makes sense, I turned it into a separate function and committed it

File: Comp2022/src/main/deploy/2135_configuration.txt [hunk 4]
```diff
@@ -35,7 +35,8 @@
   # DTL_VertOffset2 is typically negative
   DTL_VertOffset2 = -8.34
 # Path following settings
-  DTR_RamsetePidKp = 2.0
+  DTR_RamsetePidKf = 0.0465
```
[the remark this answers is not in the document]
Comment (ngupta-22) [reply Comp2022/src/main/deploy/2135_configuration.txt:38]: I calculated kF by measuring motor L1's velocity at 75% output. I multiplied the percent output by 1023 (the Talon PIDf controller's max output) and divided by the measured velocity (in native units per 100ms) to get 0.0465. This is similar to kF values previously calculated for climber and shooter, so I am quite confident with that. 

I calculated an initial value for kP as well, based on a throttle output of 10% and the average error seen using only feed-forward gain, which was ~1000 nu/100ms. There is definitely more tuning to be done for that, which we can hopefully do on Sunday.

**Pull request merged**